### DISCUSSION TITLE
How to make like a custom action for like the whole records?

### DISCUSSION THREAD

**Discussion by iamajvillalobos** [avo-hq/avo#3257] [Help] [answered]
I've read the page at https://docs.avohq.io/3.0/actions.html but I'm not sure how to proceed with what I wanted or is it possible in Avo?

Basically, I have a page (ingredients) with a list of ingredients, I have a category filters on them to easily filter them for viewing. What I want to add is to be able to print all of the records with the category. Basically can I do an action without selecting a record?

**Comment by Paul-Bob** [comment, marked as answer]
> Basically can I do an action without selecting a record?

Yes, that is a [standalone action](https://docs.avohq.io/3.0/actions.html#standalone-actions)

Inside that action, you can introduce a select field to select the category or create several actions, one for each category.

**Reply by iamajvillalobos** [reply]
Thank you so much! I will try it this week.

**Reply by Paul-Bob** [reply]
Anytime! Let me know how it goes
